bundles/org.openhab.voice.mimictts/src/main/resources/OH-INF/i18n/mimictts_fr.properties

```diff
@@ -0,0 +1,11 @@
+voice.config.mimictts.audioVolatility.label = Volatilité Audio
```

Comment:
This is strange because this file already exists in the repo. 
Maybe because the PR was created before this bonding was added? 
Maybe you should close this PR and see what new PR will be creates by Crowdin.